sechub-shared-kernel/src/main/java/com/mercedesbenz/sechub/sharedkernel/assistant/UseCaseUserRequestsFindingExplanation.java

```diff
@@ -0,0 +1,26 @@
+package com.mercedesbenz.sechub.sharedkernel.assistant;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.mercedesbenz.sechub.sharedkernel.Step;
+import com.mercedesbenz.sechub.sharedkernel.usecases.UseCaseDefinition;
+import com.mercedesbenz.sechub.sharedkernel.usecases.UseCaseGroup;
+import com.mercedesbenz.sechub.sharedkernel.usecases.UseCaseIdentifier;
+
+/* @formatter:off */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@UseCaseDefinition(
+id = UseCaseIdentifier.UC_USER_REQUESTS_FINDING_EXPLANATION,
+group = UseCaseGroup.USER_SELF_SERVICE,
+apiName = "userRequestsFindingExplanation",
```

Comment:
We used in other open api parts/identifiers a "user$does$xyz" approach and no "user$isDoing$xyz" , so please change to:

```suggestion
apiName = "userRequestFindingExplanation",
```